pages/business/dashboard.tsx

```diff
@@ -12,6 +12,47 @@ import { AddCard } from '~ui/blocks/AddCard'
 import { BusinessDataModal } from '~ui/modals/BusinessDataModal'
 import { BusinessDeleteModal } from '~ui/modals/BusinessDeleteModal'
 
+const CompanyCard = ({ company, openModal }) => {
+  const menuText = React.useMemo(() => {
+    if (company.menuPdfLink) return 'PDF Anhang'
+
+    return company.menuLink || '–'
+  }, [company])
+
+  return (
+    <ActionCard
+      href="/business/company/[companyId]"
+      as={`/business/company/${company.id}`}
+    >
+      <ActionCard.Main
+        title={company.name}
+        subtitle={'SpeiseKarte ' + menuText}
+      />
+      <ActionCard.Actions>
+        <IconButton
+          icon={Edit}
+          color="yellow.500"
+          onClick={() =>
+            openModal('data', {
+              type: 'edit',
+              name: company.name,
+              menuLink: company.menuLink,
+              menuPdfLink: company.menuPdfLink,
+              companyId: company.id,
+            })
+          }
+          title="Ändern"
+        />
+        <IconButton
+          icon={Trash}
+          color="red.500"
+          onClick={() => openModal('delete')}
```

Comment:
It would be nicer if – instead of passing `openModal` as a prop – this component had two props for `onEdit` and `onDelete`. This way only the `DashboardPage` has the concern for handling its modals, instead of passing this concern down to other components.